### PR TITLE
Create Advanced Commands

### PR DESCRIPTION
**Pull request by ghost**
# Create Advanced Commands
A Collection of Advanced Configurable and Toggleable Commands

 - Initial Upload
 - Move commands from `DynoCC/Dyno-Custom-Commands/tree/master/Fun Commands/Mini-Games/Dyno-RPG` to `DynoCC/Dyno-Custom-Commands/tree/master/Advanced Commands/Avelon-RPG`
 - Create `avelon-rpg.txt`
 - Update `README.md`

### CODE REVIEW

**Comment by ghost** [comment]
# Note

 - Conflicts can't be resolved on my end due to the cause being that the conflicting files were removed and placed into a different folder

**Comment by Swordington** [comment]
Fun, I'll see if I can fix these on my PC.

**Comment by ghost** [comment]
> 
> 
> Fun, I'll see if I can fix these on my PC.

Personally I think all it really needs is to be force pushed and it will auto resolve them. I'm pretty sure it's because the move was done in one of the later commits to this rather than the first one, so I think it's those earlier commits conflicting with the one where the files werte moved.

**Review by ghost — COMMENTED** [review]
This one would be one of the conflicts since it was done before the locatioin change

File: Information Commands/Information Tools/8ball.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{choose:It is certain;It is decidedly so;Without a doubt;Yes definitely;You may rely on it;As I see it, yes;Most likely;Outlook good;Yes; Signs point to yes;Reply hazy try again;Ask again later;Better not tell you now;Cannot predict now;Concentrate and ask again;Don't count on it;My reply is no;My sources say no;Outlook not so good;Very doubtful}{choose1:​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​}
+{!announce {channel} {choice1}}
+{!announce {channel} {choice1}}
+{!ann$0+ounce {channel} You have to ask a question, silly!}
```
Comment (thatrelevantperson) [inline Information Commands/Information Tools/8ball.txt]: Is this intentional or a mistake?

- {!ann$0+ounce

**Comment by Swordington** [comment]
Hi, there's been no activity here for a few days, if this PR is dead no action is required and will be closed. If it isn't dead there are some things that need to be fixed (see above review by @thatrelevantperson).

File: Information Commands/Information Tools/8ball.txt (hunk 1 above)
Comment (ghost) [reply Information Commands/Information Tools/8ball.txt]: Yo soz for the late reply. That is intentional. It's using custom flags for the commands. There's specific lines of code in the setup commands that is designed to tell that part of the command code what to put for the command.

Comment (PhantomNimbi) [reply Information Commands/Information Tools/8ball.txt]: It's using custom flags for the commands =). The code above it tells that what to put for the command =)

**Comment by ghost** [comment]
Yo soz for the late reply. I've been a tad busy irl lately. A lot has been going on with the family. I responded to the message posted by  @thatrelevantperson

**Comment by ghost** [comment]
**__Update 03/09/2021__**

 - Advanced [DynoCC](https://github.com/DynoCC/Dyno-Custom-Commands) Commands moved to [Discord-Coding-Community/DCC-Commands](https://github.com/Discord-Coding-Community/DCC-Commands) for ease of management.
 - New website for **Discord-Coding-Community/DCC-Commands** can be found [HERE](https://dynocc.newhorizon.dev/).
 - Will attempt to work with both [DynoCC/Dyno-Custom-Commands](https://github.com/DynoCC/Dyno-Custom-Commands)  and [StrandCC/Strand-Custom-Commands](/github.com/StrandCC/Strand-Custom-Commands) if they are interested.
 - Repo moved due to the recent updates causing merge conflicts due to the change in file location(s).
 This will make it easier for me to fix any issues that arise and will also allow me to work on them on my own time
 without worry of accidentally missing a response.
 - Go ahead and close this Pull Request

**Pull request closed without merging**